packages/components/src/mobile/bottom-sheet/missing-cell.native.js

```diff
@@ -0,0 +1,45 @@
+
+/**
+ * WordPress dependencies
+ */
+import { withPreferredColorScheme } from '@wordpress/compose';
+import { Icon } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+/**
+ * Internal dependencies
+ */
+import Cell from './cell';
+import styles from './styles.scss';
+
+function BottomSheetMissingCell( props ) {
+	const {
+		getStylesFromColorScheme,
+		onChange,
+		...cellProps
+	} = props;
+
+	const infoIconStyle = getStylesFromColorScheme( styles.infoIcon, styles.infoIconDark );
+
+	return (
+		<Cell
+			{ ...cellProps }
+			onPress={ onChange }
```

Comment:
Why isn't `onChange` here called `onPress`?

Reply:
Going to rename it.

Reply:
Current component is only displaying text passed as a `label`, there is no more `onPress` functionality.